spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/ehcache/CacheManagerTest.java

```diff
@@ -0,0 +1,47 @@
+package org.springframework.boot.autoconfigure.ehcache;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.boot.autoconfigure.PropertyPlaceholderAutoConfiguration;
+import org.springframework.boot.test.EnvironmentTestUtils;
+import org.springframework.cache.ehcache.EhCacheCacheManager;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for {@link org.springframework.cache.CacheManager}.
+ *
+ * @author Eddú Meléndez
+ */
+public class CacheManagerTest {
```

Comment:
I am confused. What's the specific purpose of this class compared to the other one?